lib/ruby-push-notifications/mpns.rb

```diff
@@ -0,0 +1,5 @@
+require 'ruby-push-notifications/mpns/mpns_connection'
+require 'ruby-push-notifications/mpns/mpns_notification'
+require 'ruby-push-notifications/mpns/mpns_pusher'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.